tests/test-privacy.php

```diff
@@ -51,4 +51,18 @@ public function test_addPrivacyPolicyContent() {
 		$this->assertTrue( $result );
 	}
 
+	/**
+	 * @group privacy
+	 */
+	public function test_namespace() {
+		$last_updated_before = get_blog_details()->last_updated;
+		update_site_option( 'pressbooks_sharingandprivacy_options', [ 'network_directory_excluded' => 0 ] );
+		add_action( 'admin_init', '\Pressbooks\Admin\Laf\privacy_settings_init' );
+		@do_action( 'admin_init' );
+		do_action( 'update_option_pb_book_directory_excluded', '0', '1' );
+		$last_updated_after = get_blog_details()->last_updated;
+
```

Comment:
is it possible to check if the exclude is updated?

Reply:
I added a new assertion to test the meta value :+1: